go.mod

```diff
@@ -2,13 +2,16 @@ module github.com/LF-Engineering/insights-datasource-git
 
 go 1.15
 
+replace github.com/LF-Engineering/lfx-event-schema => /Users/swapnilwalke/Documents/go-workspace/LF-Engineering/lfx-event-schema
```

Comment:
This won't work for me locally, right? Why do we need this?

Reply:
yes, this wont work. This was done for local testing as the changes I have added in this pull request are dependent upon the https://github.com/LF-Engineering/lfx-event-schema/pull/62 request that is yet to be merged. So to test the changes had to do it. Will need to update this as mentioned pull request gets merged. Its not needed in deployment.